tests/functional/test_client_class_names.py

```diff
@@ -69,13 +67,10 @@
 }
 
 
-def test_client_has_correct_class_name():
-    session = botocore.session.get_session()
-    for service_name in SERVICE_TO_CLASS_NAME:
-        client = session.create_client(service_name, REGION)
-        yield (_assert_class_name_matches_ref_class_name, client,
-               SERVICE_TO_CLASS_NAME[service_name])
-
-
-def _assert_class_name_matches_ref_class_name(client, ref_class_name):
-    assert_equal(client.__class__.__name__, ref_class_name)
+class TestClientClassNames(unittest.TestCase):
+    def test_client_has_correct_class_name(self):
+        session = botocore.session.get_session()
+        for service_name in SERVICE_TO_CLASS_NAME:
+            client = session.create_client(service_name, REGION)
+            self.assertEqual(client.__class__.__name__,
```

Comment:
Well, I didn’t want to change `unittest` based tests, pytest support them without problems anyway, and they don’t create any additional dependency, so if anybody wants to port those to pytest, they can do it in the next iteration.

Reply:
`assert` isn't specific to pytest, but would allow us to be consistent and avoid any dependency, including unittest. I agree this can be addressed later though.